DoLahTest/CalendarUnitTest.cpp

```diff
@@ -11,53 +13,79 @@ namespace DoLahTest
 {		
 	TEST_CLASS(CALENDAR_TESTER) {
 	private:
-		DoLah::Calendar testCal;
-		std::vector<DoLah::AbstractTask*> testVector;
-
-	public:
-		TEST_METHOD_INITIALIZE(Startup) {
-			DoLah::FloatingTask* task1 = new DoLah::FloatingTask;
-			task1->setName("CS2103 homework");
-			DoLah::FloatingTask* task2 = new DoLah::FloatingTask;
-			task2->setName("CS2101 homework");
-			DoLah::FloatingTask* task3 = new DoLah::FloatingTask;
-			task3->setName("singing practice");
-
-			testVector.push_back(task1);
-			testVector.push_back(task2);
-			testVector.push_back(task3);
-
-			testCal = DoLah::Calendar();
-
-			for (int i = 0; i < testVector.size(); i++) {
-				testCal.addTask(testVector[i]);
-			}
-		}
+
+    public:
+        TEST_METHOD(AddTask) {
+            //Arrange
+            DoLah::Calendar testCal = DoLah::Calendar();
+            DoLah::FloatingTask* task = TaskBuilder::buildFloatingTask();
+
+            //Act
+            testCal.addTask(task);
+
+            //Assert
+            Assert::AreEqual(testCal.getTaskList().size(), (size_t) 1); 
```

Comment:
@bsmmoon @starkipraggy Please note that the assertion is wrong here.
The expected value is `1` instead of the `calendar.size()`